python/py-spiceypy/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
+
+github.setup	AndrewAnnex SpiceyPy v3.1.1 
+version	3.1.1	
```

Comment:
Checksums are the same. Is that expected behavior?

Reply:
not really... did you do a `port clean --all py*-spiceypy` before generating the checksums again (otherwise it might use the distfile you already downloaded and that checksum will be of course the same).

Reply:
Still the same checksum.

Reply:
you didn't add the line `github.tarball_from archive` as I suggested, so then of course the `checksums` will not change...

Reply:
Well, that explains it. Sorry. Another try. We're getting there. :)